kernels/builders/heuristic_spatial_array.h

```diff
@@ -283,11 +283,9 @@ namespace embree
 
                 if (likely(splits <= 1)) continue; /* todo: does this ever happen ? */
 
-                //int bin0 = split.mapping.bin(prims0[i].lower)[split.dim];
```

Comment:
I'd like to understand why this change was made a couple of years ago. Ultimately it was the source of our issue here.

Reply:
@cbenthin @freibold re https://github.com/RenderKit/embree/commit/190a0dc5f757f5e93a94851938679e089bc449f4